### PR TITLE
update on features/domais

### PR DESCRIPTION
**Pull request by LaureBro**
Hello, I did some cleaning on the features and domains listed in the updates.

### CODE REVIEW

File: content/updates/2020-06/2-option-screen-revamp.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ pim_announcement_audience:
 ---
 
 # Better manage your attribute options
-::: meta-data type="Improvement" features="Attribute Options, Productivity, Data Governance" available="late June" link-to-doc="../articles/manage-your-attributes.html#manage-simple-and-multi-selects-attribute-options"
+::: meta-data type="Improvement" features="Data Governance" available="late June" link-to-doc="../articles/manage-your-attributes.html#manage-simple-and-multi-selects-attribute-options"
```
Comment (NolwennP) [inline content/updates/2020-06/2-option-screen-revamp.md:10]: I would keep "Productivity" for this one.

Comment (LaureBro) [inline content/updates/2020-06/2-option-screen-revamp.md:10]:
```suggestion
::: meta-data type="Improvement" features="Productivity, Data Governance" available="late June" link-to-doc="../articles/manage-your-attributes.html#manage-simple-and-multi-selects-attribute-options"
```

Comment (LaureBro) [reply content/updates/2020-06/2-option-screen-revamp.md]: fixed :) thanks!

File: content/updates/2020-05/5-rules-engine-new-improvements.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ pim_announcement_audience:
 Small UI enhancements and a new parameter for your calculation action, those are the last improvements of the rules engine!
 
 ## New action tags for your rules
-::: meta-data type="Improvement" features="Productivity, Rules engine" available="early May" link-to-doc="../articles/manage-your-rules.html#view-all-rules"
+::: meta-data type="Improvement" features="Rules Engine, Productivity" available="early May" link-to-doc="../articles/manage-your-rules.html#view-all-rules"
```
Comment (NolwennP) [inline content/updates/2020-05/5-rules-engine-new-improvements.md:11]:
```suggestion
::: meta-data type="Improvement" features="Automation, Productivity" available="early May" link-to-doc="../articles/manage-your-rules.html#view-all-rules"
```

**Review by NolwennP — APPROVED** [review]
Ok for those news. Should we create another PR to harmonize all the other news.

**Comment by LaureBro** [comment]
> Ok for those news. Should we create another PR to harmonize all the other news @LaureBro?
Yes, we'll have to give another look to other news, specially for Automation. I'd like them to be aligned on the ones we have on the What's new page (Governance, Productivity, Collaboration, Connectivity, Data Quality, Automation, Reporting, Scalability).

**Pull request merged**